packages/multi-select-combo-box/test/readonly.test.js

```diff
@@ -192,6 +192,17 @@ describe('readonly', () => {
       expect(items[2].textContent).to.equal('lemon');
       expect(items[3].textContent).to.equal('orange');
     });
+
+    it('should render selected items in the dropdown when size is set', async () => {
+      inputElement.click();
+      // Wait for the async data provider timeout
```

Comment:
If you add `comboBox.readonly = false;` to the test, the issue is still there

Reply:
Actually, the test was setting wrong `size` value - 5 with only 4 items in the list. 
When using correct size, it works fine. Updated the test to do this.